app/web/src/components/ManagementRunPrototype.vue

```diff
@@ -129,6 +129,9 @@ const runPrototype = async (viewId: ViewId) => {
   );
 
   if (result.result.success) {
+    // TODO(WENDY) - for template functions, select the returned componentIds
+    // modelingEventBus.emit("setSelection", componentIds);
```

Comment:
Does this need to be done? Also, I dont think we need the event bus... its just an action in the view store now...